modules/preview-react/table/lib/TableCell.tsx

```diff
@@ -14,7 +14,7 @@ export const TableCell = createComponent('td')({
         backgroundColor="frenchVanilla100"
         borderBottom={`1px solid ${colors.soap400}`}
         gridTemplateColumns="1fr"
-        justifyContent="left"
+        justifyContent="flex-start"
```

Comment:
Nit: This works, but the component is `display: grid`, not `flex`.
```suggestion
        justifyContent="start"
```